test/MarketMakerProxy.t.sol

```diff
@@ -0,0 +1,228 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "contracts/MarketMakerProxy.sol";
+import "test/utils/BalanceSnapshot.sol";
+import "test/utils/StrategySharedSetup.sol";
+import "test/mocks/MockERC1271Wallet.sol";
+
+contract MarketMakerProxyTest is StrategySharedSetup {
```

Comment:
MMP test seems not so relevant to other tokenlon infra contracts? Not sure if it is required to inherit `StrategySharedSetup` since it would cost to setup many other contracts that MMP test doesn't need.

Reply:
I was too lazy...fixed in a3804ee.